content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -71,7 +71,7 @@ The default value for `operator` is `Equal`.
 A toleration "matches" a taint if the keys are the same and the effects are the same, and:
 
 * the `operator` is `Exists` (in which case no `value` should be specified), or
-* the `operator` is `Equal` and the `value`s are equal.
+* the `operator` is `Equal` and the `values` are equal.
```

Comment:
```suggestion
* the `operator` is `Equal` and the values should be equal.
```
How about using a more descriptive expression for this?

Reply:
please check my latest commit

Reply:
lgtm, thanks.